share/completions/solc.fish

```diff
@@ -0,0 +1,83 @@
+# General Information
+complete -c solc -l help -d "Show help message and exit."
+complete -c solc -l version -d "Show version and exit."
+complete -c solc -l license -d "Show licensing information and exit."
+complete -c solc -l input-file -d "input file"
+
+# Input Options
+complete -c solc -l base-path -d "Use the given path as the root of the source tree instead of the root of the filesystem."
+complete -c solc -l include-path -d "Make an additional source directory available to the default import callback. Use this option if you want to import contracts whose location is not fixed in relation to your main source tree, e.g. third-party libraries installed using a package manager. Can be used multiple times. Can only be used if base path has a non-empty value."
+complete -c solc -l allow-paths -d "Allow a given path for imports. A list of paths can be supplied by separating them with a comma."
+complete -c solc -l ignore-missing -d "Ignore missing files."
+complete -c solc -l error-recovery -d "Enables additional parser error recovery."
+
+# Output Options
+complete -c solc -s o -l output-dir -d "If given, creates one file per component and contract/file at the specified directory."
+complete -c solc -l overwrite -d " Overwrite existing files (used together with -o)."
+complete -c solc -l evm-version -d "Select desired EVM version. Either homestead, tangerineWhistle, spuriousDragon, byzantium, constantinople, petersburg, istanbul, berlin or london."
```

Comment:
we can tell fish to suggest these:

```suggestion
complete -c solc -l evm-version -d "Select desired EVM version" -rfa "homestead tangerineWhistle spuriousDragon byzantium constantinople petersburg istanbul berlin london"
```

though it would much better if we can ask for the versions dynamically, since it's bound to go stale